qemu/tests/cfg/cpu_info_check.cfg

```diff
@@ -0,0 +1,14 @@
+- cpu_info_check:
+    type = cpu_info_check
+    start_vm = no
+    remove_list = ''
+    cpu_model_2_12_0 = 'Skylake-Client Skylake-Server'
+    cpu_model_3_1_0 = 'Cascadelake-Server Icelake-Client Icelake-Server KnightsMill'
+    cpu_model_8_2 = 'Cascadelake-Server-noTSX Icelake-Client-noTSX Icelake-Server-noTSX'
+    cpu_model_8_2 += ' Skylake-Client-noTSX-IBRS Skylake-Server-noTSX-IBRS'
+    cpu_model_8_3 = 'Cooperlake EPYC-Rome'
+    cpu_model_8 = ${cpu_model_8_2_0} + ${cpu_model_8_3_0}
+    Host_RHEL.m8.u2, Host_RHEL.m8.u1, Host_RHEL.m8.u0:
+        remove_list = cpu_model_8_3
+    Host_RHEL.m7, Host_RHEL.m6:
+        remove_list = cpu_model_8
```

Comment:
RHEL7 host only has  `cpu_model_2_12_0`, so we also need to remove `cpu_model_3_1_0`.

Reply:
Seems we should change the Line 62 because it is related to the host qemu version.

Reply:
@nanliu-r Let me describe it: For host below rhel8(rhel7 and rhel6), first remove cpu_model_8, then according to qemu version, remove cpu_model_3_1_0 when it is below qemu3.1.0, remove cpu_model_2_12_0 when it is below qemu2.1.0.
So on rhel7 host, both cpu_model_8 and cpu_model_3_1_0 will be removed with current implementation. Does it not already satisfy the test requirement?

Reply:
> @nanliu-r Let me describe it: For host below rhel8(rhel7 and rhel6), first remove cpu_model_8, then according to qemu version, remove cpu_model_3_1_0 when it is below qemu3.1.0, remove cpu_model_2_12_0 when it is below qemu2.1.0.
> So on rhel7 host, both cpu_model_8 and cpu_model_3_1_0 will be removed with current implementation. Does it not already satisfy the test requirement?

Hi， seems this is a format issue, the logic is right...
```
04:42:45 DEBUG| [stdout] ====== model list
04:42:45 DEBUG| [stdout]

04:42:45 DEBUG| [stdout] ['cpu_model_8_2_0', '+', 'cpu_model_8_3_0']
04:42:45 DEBUG| [stdout]

04:42:45 WARNI| The model to be removed is not in the list: cpu_model_8_2_0
04:42:45 WARNI| The model to be removed is not in the list: +

```

Reply:
@nanliu-r Ah yes, it should be `remove_list = ${cpu_model_8}` instead of `remove_list = cpu_model_8`, updating. Thanks.

Reply:
And the name should be `cpu_model_8_2` instead of `cpu_model_8_2_0`, just updated.